src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -49,12 +59,13 @@ export const InlineEditExample = {
       props: { EuiInlineEditText },
     },
     {
-      title: 'InlineEditTitle',
+      title: 'Display and edit headers and titles',
```

Comment:
```suggestion
      title: 'Display and edit headings and titles',
```

semantic nit: `headers` (containers/layout) have a different meaning in HTML than `headings` (titles/text) - we should likely use the most accurate terminology